lib/Doctrine/DBAL/Platforms/MySqlPlatform.php

```diff
@@ -973,7 +973,7 @@ public function getDropIndexSQL($index, $table=null)
             return $this->getDropPrimaryKeySQL($table);
         }
 
-        return 'DROP INDEX ' . $indexName . ' ON ' . $table;
+        return 'DROP INDEX ' . $this->getIdentifierQuoteCharacter() . $indexName . $this->getIdentifierQuoteCharacter() . ' ON ' . $table;
```

Comment:
Why do you prepend/append? Use [`$this->quoteIdentifier()`](https://github.com/gabrielrincon/dbal/blob/dca8ab3c226c3166c96452d38a4eff333e3d4b51/lib/Doctrine/DBAL/Platforms/AbstractPlatform.php#L1860)